File: mycollab-mobile/src/main/scala/com/mycollab/mobile/module/crm/events/OpportunityEvent.scala
package com.mycollab.mobile.module.crm.events

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd
  * @since 5.4.9
  */
object OpportunityEvent {
  class Save(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class Search(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoList(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoAdd(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoEdit(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoRead(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

  class GotoRelatedItems(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
}

File: mycollab-mobile/src/main/java/com/mycollab/mobile/module/crm/event/OpportunityEvent.kt
package com.mycollab.mobile.module.crm.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class OpportunityEvent {
    class Save(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class Search(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoList(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoAdd(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoEdit(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoRead(source: Any, data: Any?) : ApplicationEvent(source, data) 

    class GotoRelatedItems(source: Any, data: Any?) : ApplicationEvent(source, data) 
}